app/views/spree/shared/_cart_box.html.erb

```diff
@@ -0,0 +1,216 @@
+    const vendorBox = $(item).parents('.sip-box').first();
+    const vendorId = $(vendorBox).data('vendor-id');
+
+    const renderAlignShipmentNum = () => {
+      let allBoxes = $('.sip-box').not('.deleted');
+      $.each(allBoxes, function (index, el) {
+        let lis = $(el).find('.line-item');
+        let totalPrice = 0;
+        let totalCount = 0;
+        let displayPrice = '';
+
+        $.each(lis, function (index, liEl) {
+          let {liCount,liPrice,liCurrency} = boxSingleItem(liEl);
+
+          totalPrice += (liCount * liPrice);
+          totalCount += liCount;
+          liCurrency = liCurrency.replace(/,/, "");
+
+          if (lis.length === index + 1) {
+            if (liCurrency == "$"){
+              displayPrice =  liCurrency + ' ' + totalPrice.toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+            }else{
+              displayPrice = totalPrice.toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ",") + ' ' + liCurrency;
+            }
+          }
+        })
+      })
+    }
+
+    sum = 0;
+    Object(order).line_items && order.line_items.forEach((li) => {
+      if (parseInt(li.variant.vendor_id) === parseInt(vendorId)) {
+        let cost = li.display_amount;
+        let itemCount = li.quantity;
+        renderQtyLabels(itemCount, li);
+        renderItemTotalPrice(cost, li);
+      }
+      let cost = li.display_amount;
+      this.renderItemTotalPrice(cost, li);
+      sum += li.quantity;
+      $('.cart-item-count').html(sum);
+    });
+
+    renderAlignShipmentNum();
+    this.updateSummaryGeneral(order);
+
+  };
+
+  this.renderItemTotalPrice = function (cost, li) {
+    $(".js-lineitem-total-" + li.id ).html(cost);
+  }
+
+  this.updateSummaryGeneral = function (order) {
+    $('.js-order-total').html(order.display_total);
+    $('.js-product-total').html(order.display_item_total)
+  }
+
+  this.addItem = function (e) {
+    e.preventDefault();
+    $qtyEl = $(this).siblings('.line_item_quantity').first();
+    let currentCount = parseInt($qtyEl.val());
+    let newCount = currentCount + 1;
+    $qtyEl.val(newCount);
+    that.updateItem(this, 'add', newCount);
+  };
+
+  this.subItem = function (e) {
+    e.preventDefault();
+    $qtyEl = $(this).siblings('.line_item_quantity').first();
+    let currentCount = parseInt($qtyEl.val());
+    let newCount = currentCount - 1;
+    $qtyEl.val(newCount);
+    that.updateItem(this, 'sub', newCount);
+  };
+
+  this.removeItemFromPopup = function(e){
+    e.preventDefault();
+
+    const lineItemId = $(this).data('line-item-id');
+    that.updateItem($('.js-line-item-delete-btn-'+lineItemId), 'sub', 0);
+
+  }
+  this.updatePopup = function(order){
+    $('.js-popup-cart-total').html(order.display_item_total)
+  }
+  $('body').on('click', '.popup-cart-delete', this.removeItemFromPopup);
+  $('body').on('click', '.jsAddItem', this.addItem);
+  $('body').on('click', '.jsSubItem', this.subItem);
+}
+
+</script>
+<script>
```

Comment:
dnt move this js . As it needs to be here to call the functionality. @hamzanaveed9